### ISSUE TITLE
dpkg: packages from 3rd party repos missing maintainer warnings

### ISSUE THREAD

**Issue by Tweaker177** [ElectraJailbreak/electratools#20]
dpkg: warning: parsing file 'var/lib/dpkg/status' near line xxxx package 'any 3rd party repo package': missing maintainer

Happens for all yourepo packages installed, as well as another repo I heavily use, and I assume any packages installed that aren't from a default repo.  Just from 8 tweaks I installed from my personal repos plus a few others have a full page of errors every time I install any package now.   Sorry if issue is in wrong spot wasn't sure if this should be under Cydia repo or this one.

**Comment by dallasgroot** [comment]
There is a work around by adding these lines, just copy the maintainer line from somewhere else and paste it the places where its missing (cydia will tell you which ones) this will work until you update the package of that specific one that isn't working....)

**Comment by Tweaker177** [comment]
Yeah, the workaround was easy enough to figure out b4 reading the response, just thought most end users wouldn't know what and where to change. I was contemplating making a tweak that reads the package lists and alters them by just copying author as maintainer if maintainer doesn't exist before the next occurrence of author shows up. 

Just figured it was ez enough of a fix that rather then make a new tweak (or update my FormulaOneOne tweak with workaround as extra feature) that I'd mark it as an issue if you guys wanted to fix it as part of the jailbreak or as part of one of the core packages.

I'm a newbie jailbreak developer so figured changing core files like that was something I didn't want to risk messing up if it didn't work for someone. Also just don't have much free time lately my day job is killing me.

Thx for getting back to me tho, and keep up the good work. Maybe if I get a chance and write some code to implement the workaround programmatically I'll just do a pull on the Electra github and more experienced ppl can decide to use it, alter it, or just scrap it. I know there's much more important issues then that so I'd understand if it's not viewed as a priority.

Take care,
-Brian / i0stweak3r
